proactive/ProactiveBuilder.py

```diff
@@ -0,0 +1,119 @@
+from .ProactiveTask import *
```

Comment:
`import *` can lead to problems (overriding stuff from other modules) depending on the order of the code calling your module proactive.

Reply:
hello @paraita , in our case this kind of situation is not very impactful because all classes begin with **ProactiveXXX** and usually, in python, we can have more than 1 class per file (like C/C++), so it is a way to not specify classes by their name every time...in C/C++ we have *namespace* to help us on that, but in python, if we have classes conflict by name, we can do `import *** as blablaba` ...and we use `blablabla` as reference. In fact, the end-users will not use this type of sintax `from .ProactiveTask import *`, they will use `import proactive` or `import proactive as pa` and do `pa.blablabla`...

Reply:
non non non... i said that usually the python users, when they are using third-party packages, usually they import packages like `import proactive` like this:
```
import proactive

print("Logging on proactive-server...")
proactive_host = '10.220.248.146'
proactive_port = '8080'
proactive_url  = "http://"+proactive_host+":"+proactive_port
print("Connecting on: " + proactive_url)
gateway = proactive.ProActiveGateway(proactive_url)

gateway.connect("admin", "proactive")
print("Connected")

print("Creating a proactive python task...")
pythonTask = gateway.createPythonTask()
pythonTask.setTaskName("SimplePythonTask")
pythonTask.setTaskImplementation("""print("Hello world!")""")
#pythonTask.setTaskImplementationFromFile("_demo_script.py")
#pythonTask.setTaskImplementationFromLambdaFunction(lambda: 88 - 20 * 10)
#pythonTask.addGenericInformation("PYTHON_COMMAND", "/usr/bin/python3")

print("Creating a proactive job...")
myJob = gateway.createJob()
myJob.setJobName("SimplePythonJob")
myJob.addTask(pythonTask)

print("Submitting the job to the proactive scheduler...")
job_id = gateway.submitJob(myJob)
print("job_id: " + str(job_id))

## disconnect
print("Disconnecting")
gateway.disconnect()
print("Disconnected")
```
if they want to do `from .ProactiveTask import *` they can, but the ProactiveTask.py file need to be in the same folder, and this is not the case, because the our proactive python connector is a python package, they will not work `in the` package, but `calling` the package from they workspace.